nvm.sh

```diff
@@ -104,7 +104,7 @@ nvm_download() {
     if nvm_curl_use_compression; then
       CURL_COMPRESSED_FLAG="--compressed"
     fi
-    curl --fail ${CURL_COMPRESSED_FLAG:-} -q "$@"
+    command curl --fail ${CURL_COMPRESSED_FLAG:-} -q "$@"
```

Comment:
i feel like i left this one as-is for some reason; can you try git blame-ing this line?

(i think there's probably people that alias or shadow `curl`, and this would break them)

Reply:
I traced it, at least can trace back to #452, didn't see that reason yet, what do you think?

Reply:
Looks like it'll break our tests, as they are shadow `curl` in fact ... maybe we should use a stronger mock method like to temporarily override the `curl` binary or something like that? It'll increase the complexity, but I'm not sure if it'll be good to not use the `command` prefix just because we want simpler tests.

Reply:
Let's skip the prefix on `curl` for now; we can address that in a future PR.

Reply:
Actually, we have `command` prefix in `nvm debug`, as it's not tested, maybe we can still keep the one on the curl not tested?

Reply:
Since some tests require `curl` be un-prefixed, i think all curls should be unprefixed, including in `nvm debug`.

Reply:
Okay, I think we should separate that change another place, right?

Reply:
Reverted that prefix added here for curl.